core/services/network_checker.rb

```diff
@@ -23,11 +23,14 @@ def self.resources_available?(machine_configurator, machine, logger)
            else
              :wget
            end
-    result = random_resources(resources).all? do |resource|
-      check_resource?(tool, machine_configurator, machine, resource, logger)
+    table = {}
+    resources.each do |resource|
+      check_resource?(tool, machine_configurator, machine, resource, logger, table)
```

Comment:
You must fill the table in current method, not inside the `check_resource?` method.